### DISCUSSION TITLE
Whether or not displaying avif format images is not supported

### DISCUSSION THREAD

**Discussion by littletoxic** [TriliumNext/Trilium#6049] [Ideas]
![image](https://github.com/user-attachments/assets/c17deef1-ab35-4f54-b3da-0f32e54cd279)
When I paste a picture in avif format, the page appears like this

**Comment by Nriver** [comment]
The only way to display `avif` images in Trilium is to compress them into a `.zip` file and import that. After importing, I can then copy the image reference. Directly pasting the `avif` images results in broken links.

Instead, I use `.webp` images, which work well in Trilium and offer a good compression rate.
